### PR TITLE
added hw-19-1

### PR DESCRIPTION
**Pull request by kenu21**

### CODE REVIEW

File: logs/internetshop.log [hunk 1]
```diff
@@ -1,23 +1,11 @@
-2019-09-20 19:08:20 INFO  InjectInitializer:15 - Dependency injection started...
```
Comment (Sofasmile) [inline logs/internetshop.log:1]: you may remove logs from repository)

File: src/main/java/mate/academy/internetshop/controller/LogoutController.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        HttpSession session = req.getSession(true);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/controller/LogoutController.java]: can we simplify?
HttpSession session = req.getSession(true);
        session.setAttribute("userId", "");
        session.invalidate();
        session = null;
-->
req.getSession().invalidate();

File: src/main/java/mate/academy/internetshop/model/Role.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package mate.academy.internetshop.model;
+
+public class Role {
+    private static long idGenerator = 0;
+    private final Long id;
+    RoleName roleName;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/model/Role.java]: add access modifier

File: src/main/java/mate/academy/internetshop/web/filters/AuthorizationFilter.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.service.UserService;
+
+public class AuthorizationFilter implements Filter {
+    public static final String EMPTY_STRING = "";
+
+    @Inject
+    private static UserService userService;
+
+    private Map<String, Role.RoleName> protectedUrls = new HashMap<>();
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/web/filters/AuthorizationFilter.java:31]: how about other urls?

File: src/main/java/mate/academy/internetshop/controller/LogoutController.java (hunk 2 above)
Comment (kenu21) [reply src/main/java/mate/academy/internetshop/controller/LogoutController.java]: OK.

File: src/main/java/mate/academy/internetshop/model/Role.java (hunk 3 above)
Comment (kenu21) [reply src/main/java/mate/academy/internetshop/model/Role.java]: OK.

File: src/main/java/mate/academy/internetshop/web/filters/AuthorizationFilter.java (hunk 4 above)
Comment (kenu21) [reply src/main/java/mate/academy/internetshop/web/filters/AuthorizationFilter.java:31]: OK.

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**